templates/rhpam71-kieserver-externaldb.yaml

```diff
@@ -610,7 +688,7 @@ objects:
           - name: KIE_SERVER_CONTROLLER_SERVICE
             value: "${KIE_SERVER_CONTROLLER_SERVICE}"
           - name: KIE_SERVER_CONTROLLER_PROTOCOL
-            value: "ws"
+            value: "${KIE_SERVER_CONTROLLER_PROTOCOL}"
```

Comment:
This should be hardcoded as "ws" at this point. I think your rebase re-introduced the parameter.